react-app/src/native/components/leagues/scheduleWeek.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import LeagueHelpers from '../../../utils/leaguehelpers';
+
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class ScheduleWeek extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    /**
+     * need to sort games by the location
+     * optimize some code as it is kinda slow when loading a league
+     * add locations names to schedule
+     * add bold to the times
+     * take care of unscheduled games
+     */
+
+    render() {
+
+        const { schedule, league } = this.props;
+
+        const flexArr = [4, 2, 2, 2];
+        const weekTable = {
+            header: ['Field', 'Dark', '', 'White'],
+            data: [],
+        }
+
+        //week1: { date: {}, matchTimes: [], matches: [{venue, time, team1, team2}]},
+        let prevTime = '';
+        schedule.matches.map((match, i) => {
+            if(prevTime != match.time){
+                weekTable.data.push([LeagueHelpers.convertMatchTime(match.time), '','','']);
+            }
+            weekTable.data.push([match.venue, LeagueHelpers.getNumInLeague(league, match.team1), 'vs', LeagueHelpers.getNumInLeague(league, match.team2)]);
```

Comment:
Instead of team #'s can you put the team names into the schedule? It might looks busy but I think the way you have the schedules set up it makes more sense than the #'s.